README.md

```diff
@@ -334,6 +334,22 @@ If you want to use a different default formatter, you can add a `<format **>` (o
 
 Defines which formatter to use.
 
+### Net::HTTP::Persistent parameters (optional)
+
+The following parameters can be used for tuning HTTP connections
+
+#### idle_timeout (integer)
+
+If a connection has not been used for this number of seconds it will automatically be reset upon the next use to avoid attempting to send to a closed connection. nil means no timeout.  The default is 5 seconds.
```

Comment:
Please follow the `Default value` format from the other parameters in the README.